benchmarks/gabriel-benchmarks/stak.lisp

```diff
@@ -0,0 +1,75 @@
+;;;; gabriel-benchmarks/stak.lisp
+;;;;
+;;;;
+
+(cl:in-package #:coalton-benchmarks)
+
+(define-benchmark stak ()
+  (declare (optimize speed))
+  (loop :repeat 1000
+        :do (with-benchmark-sampling
+              (coalton:coalton (coalton-benchmarks/native:stak 18 12 6))))
```

Comment:
Add type annotations and then you can call the function directly.

Reply:
Added!